### PR TITLE
Merging Bimonthly into Monthly, excluding sarek and rnaseq updates

### PR DESCRIPTION
**Pull request by slohse**
This PR merges large parts of the bimonthly branch into monthly, excluding the updated versions of rnaseq and sarek, which have not been validated, yet.

### CODE REVIEW

File: roles/nf-core/defaults/main.yml [hunk 1]
```diff
@@ -20,13 +20,15 @@ pipelines:
     release: "{{ sarek_tag }}"
     container_dir: "{{ biocontainers_dirname }}"
     pipeline_specific_containers:
-      - sarekvep
-      - sareksnpeff
+      - vep-106.1
+      - snpeff-5.1
```
Comment (b97pla) [inline roles/nf-core/defaults/main.yml:24]: Do you know how these versions ☝️ relate to the sarek version? I would be inclined to say that these should not be changed right now either. And neither should the addition of reference genomes below 👇

**Review by ssjunnebo — APPROVED** [review]

**Review by b97pla — CHANGES_REQUESTED** [review]
:+1: looks good but unless there is a specific reason for changing the sarek pipeline-specific containers, I think they should be the same as in the current production release

File: roles/nf-core/defaults/main.yml (hunk 1 above)
Comment (slohse) [reply roles/nf-core/defaults/main.yml]: Good catch! I excluded this commit now, too.

[the remark this answers is not in the document]
Comment (slohse) [reply roles/nf-core/defaults/main.yml]: Yep, excluded that commit too, now.

**Pull request merged**